src/coreclr/vm/jithelpers.cpp

```diff
@@ -5671,6 +5671,34 @@ HCIMPLEND_RAW
 EXTERN_C void JIT_ValidateIndirectCall();
 EXTERN_C void JIT_DispatchIndirectCall();
 
+HCIMPL2_VV(void*, JIT_DispatchVirtualStatic, MethodTable *constrainedType, MethodDesc *interfaceMethod)
```

Comment:
The section of the file is for `INTEROP HELPERs`. This should be moved next to ` JIT_VirtualFunctionPointer`.

Reply:
Thanks for pointing that out, moved according to your suggestion.